llvm/lib/Target/SPIRV/SPIRVPrepareFunctions.cpp

```diff
@@ -380,8 +432,11 @@ SPIRVPrepareFunctions::removeAggregateTypesFromSignature(Function *F) {
 
 bool SPIRVPrepareFunctions::runOnModule(Module &M) {
   bool Changed = false;
-  for (Function &F : M)
+  for (Function &F : M) {
     Changed |= substituteIntrinsicCalls(&F);
+    Changed |= stripConvergenceOperands(&F);
+    Changed |= stripConvergenceIntrinsics(&F);
```

Comment:
Shouldn't we combine these two functions into one? They both have the same structure, do similar things, and called only once and together.

Reply:
Sure, done!